### ISSUE TITLE
USER ISSUE: Cant connect to "single player" game

### ISSUE THREAD

**Issue by boothe89** [StrangeLoopGames/EcoIssues#8315]
**Version:** 0.7.5.1 beta

**Steps to Reproduce:**
Start Eco
Load Single Player Game	

**Expected behavior:**
Load the Map

**Actual behavior:**
The following error message popped up:
```
<size=150%>Connection Failed</size>

Failed to establish connection - no response from remote host
```

**Comment by boothe89** [comment]
I'm noticing that the EcoServer Process is not starting when I load up a Single Player game.  It just started happening all of a sudden.  Server was on for 5 days (I left my computer running) and I go to mine some stone, and the server shot me out of my mine, and i glitched all over the place.  So I shut down my computer and apon the restart I keep getting the error message stated above.

Any help is greatly appreciated.

**Comment by pamduncan** [comment]
Can you delete steamapps/common/Eco/Eco_data/Server/EcoMods.dll with your server off and then try to load your world again.  If you have been in this world before that should fix it so you can now log in.

If this is a new world you would need to start the servers outside the client and let it create a new game which it will name Game.  Then once its up and running outside the client go into Eco and choose 'join' and find your server listed as a LAN server and try and log in.  You may have to try a couple of times but it should write your player profile in a couple of tries and let you in the world.  Then you can close down the server and use the Eco game to 'load' the world called "Game" that you were just on.

**Comment by boothe89** [comment]
Deleting the ecomods.dll did the trick.

Thank you for your input.

**Comment by SeanAtr0n** [comment]
should be fixed in the 7.5.1 hotfix today
